sdks/python/apache_beam/examples/complete/juliaset/juliaset/juliaset.py

```diff
@@ -51,14 +51,15 @@ def point_set(n):
 
   julia_set_colors = (pipeline
                       | beam.Create('add points', point_set(n))
-                      | beam.Map(get_julia_set_point_color, c, n, max_iterations))
+                      | beam.Map(
+                          get_julia_set_point_color, c, n, max_iterations))
 
   return julia_set_colors
 
 
 def generate_julia_set_visualization(data, n, max_iterations):
   """Generate the pixel matrix for rendering the julia set as an image."""
-  import numpy as np  # pylint: disable=g-import-not-at-top
+  import numpy as np  # pylint: disable=wrong-import-order
```

Comment:
It does not hit, but you are right it should hit. This might be related to pylint displaying only one of the two violations happening here:

wrong-import-order: numpy is after apache_beam (It should be third_party first then local)
wrong-import-position: Import is not at the top of the file.

Nevertheless, I updated pylint disable statements to disable for both import related warnings when both are present.